modules/gdnative/gdnative/packed_arrays.cpp

```diff
@@ -30,882 +30,101 @@
 
 #include "gdnative/packed_arrays.h"
 
-#include "core/variant/array.h"
-
 #include "core/variant/variant.h"
 
-#include "core/math/color.h"
 #include "core/math/vector2.h"
-#include "core/math/vector3.h"
+
+static_assert(sizeof(godot_packed_byte_array) == sizeof(PackedByteArray), "PackedByteArray size mismatch");
+static_assert(sizeof(godot_packed_int32_array) == sizeof(PackedInt32Array), "PackedInt32Array size mismatch");
+static_assert(sizeof(godot_packed_int64_array) == sizeof(PackedInt64Array), "PackedInt64Array size mismatch");
+static_assert(sizeof(godot_packed_float32_array) == sizeof(PackedFloat32Array), "PackedFloat32Array size mismatch");
+static_assert(sizeof(godot_packed_float64_array) == sizeof(PackedFloat64Array), "PackedFloat64Array size mismatch");
+static_assert(sizeof(godot_packed_string_array) == sizeof(PackedStringArray), "PackedStringArray size mismatch");
+static_assert(sizeof(godot_packed_vector2_array) == sizeof(PackedVector2Array), "PackedVector2Array size mismatch");
+static_assert(sizeof(godot_packed_vector2i_array) == sizeof(Vector<Vector2i>), "Vector<Vector2i> size mismatch");
+static_assert(sizeof(godot_packed_vector3_array) == sizeof(PackedVector3Array), "PackedVector3Array size mismatch");
```

Comment:
Vector3i seems missing.

Reply:
There's not really a PackedVector3iArray. I only left the Vector2i because the text server extension is using it. I guess I can add a Vector3i for completeness.